examples/imagenet/imagenet_lib.py

```diff
@@ -30,6 +30,7 @@
 import jax.numpy as jnp
 
 import tensorflow as tf
+tf.config.experimental.set_visible_devices([], "GPU")
```

Comment:
Realized after seeing some jax2tf test failures - We have to only call this after absl flag parsing / absl lib init happens so this should instead be put in "imagenet_main.py" and not a the example training lib file.

Reply:
Thanks for highlighting this. I have moved th `tf.config.experimental.set_visible_devices([], 'GPU')` into the `*_main.py` (or `train.py`) and `_lib_test.py` files (I could have put them into the `train_and_evaluate()` methods but I thought it cleaner to leave this setup to the main/test setup).